lpe/lpe.cpp

```diff
@@ -190,14 +192,19 @@ int main(int argc, char *argv[])
 			{
 				pathSeen = true;
 				path = argv[i];
-				dpPtr = strrchr(argv[i], ':');
```

Comment:
This actually has the advantage that : before the end will not affect things. Can or should we preserve that? colon is a valid filename character after all...